floodviz/static/js/hydrograph.js

```diff
@@ -1,17 +1,19 @@
-document.addEventListener("DOMContentLoaded", function (event) {
-	"use strict";
+"use strict";
 
-	var data_path = FV.hydrograph_data_path;
+FV.hydromodule = function (options) {
 
-	// Set the dimensions of the canvas / graph
+	var self = {};
 	var margin = {top: 30, right: 20, bottom: 30, left: 50};
-	var width = FV.chart_dimensions.width - margin.left - margin.right;
-	var height = FV.chart_dimensions.height - margin.top - margin.bottom;
-
-	// Set the ranges
-	var x = d3.scaleTime().range([0, width]);
-	var y = d3.scaleLog().range([height, 0]);
 
+	var height = options.height - margin.top - margin.bottom;
+	var width = options.width - margin.left - margin.right;
+	var sel_div = options.div_id;
```

Comment:
You don't really need to declare a new variable ... just use options.div_id where you are using sel_div. This makes it clearer that it's passed into the function.